openpaas-james/apps/distributed-ldap/src/main/java/com/linagora/openpaas/james/app/DistributedLdapServer.java

```diff
@@ -0,0 +1,57 @@
+package com.linagora.openpaas.james.app;
+
+import static org.apache.james.CassandraJamesServerMain.REQUIRE_TASK_MANAGER_MODULE;
+import static org.apache.james.JamesServerMain.LOGGER;
+
+import org.apache.james.CassandraRabbitMQJamesConfiguration;
+import org.apache.james.GuiceJamesServer;
+import org.apache.james.JamesServerMain;
+import org.apache.james.SearchConfiguration;
+import org.apache.james.SearchModuleChooser;
+import org.apache.james.data.LdapUsersRepositoryModule;
+import org.apache.james.modules.DistributedTaskManagerModule;
+import org.apache.james.modules.DistributedTaskSerializationModule;
+import org.apache.james.modules.blobstore.BlobStoreCacheModulesChooser;
+import org.apache.james.modules.blobstore.BlobStoreConfiguration;
+import org.apache.james.modules.blobstore.BlobStoreModulesChooser;
+import org.apache.james.modules.event.RabbitMQEventBusModule;
+import org.apache.james.modules.queue.rabbitmq.RabbitMQModule;
+import org.apache.james.modules.server.JMXServerModule;
+import org.apache.james.modules.server.RabbitMailQueueRoutesModule;
+
+import com.google.inject.Module;
+import com.google.inject.util.Modules;
+
+public class DistributedLdapServer {
+    private static final Module MODULES = Modules
+        .override(Modules
+            .override(Modules.combine(REQUIRE_TASK_MANAGER_MODULE, new DistributedTaskManagerModule()))
```

Comment:
You didn't removed the modules POP3, lmtp and manage sieve.
You can have a look at https://github.com/linagora/openpaas-james/pull/17/files#diff-c79193e1524eeba68fa23b360263485865d07c0f6d800304c22b1a5a2fec37cd 

And you can remove the resources/managesieveserver.xml